Assets/MRTK/SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -2580,8 +2580,13 @@ private void OnPointerUp(MixedRealityPointerEventData eventData)
             }
         }
 
-        private void OnPointerDown(MixedRealityPointerEventData eventData)
+        void IMixedRealityPointerHandler.OnPointerDown(MixedRealityPointerEventData eventData)
         {
+            // Other option is to check propagation phase
+            if(eventData.selectedObject == gameObject)
+            { 
+                return; 
```

Comment:
Why are you exiting early if the event is sent directly to this object? Seems counter-intuitive so an explanation may help.